TodoApp.ConsoleApp/UI/TaskOperations.cs

```diff
@@ -40,8 +43,7 @@ public static void ReadOrWrite()
 
         static void ReadTasks()
         {
-            XMLTaskReader reader = new XMLTaskReader();
-            string path = "../../tasks.xml";
+            TodoService service = new TodoService();
```

Comment:
Define and initialize the service in a single place for this class. Do not initialize it for every method.